ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/services/testing/ApiTimeService.scala

```diff
@@ -71,9 +71,6 @@ class ApiTimeService private (
 
   @SuppressWarnings(Array("org.wartremover.warts.JavaSerializable"))
   override def setTime(request: SetTimeRequest): Future[Empty] = {
-    @SuppressWarnings(Array("org.wartremover.warts.ExplicitImplicitTypes"))
-    implicit val dec = DirectExecutionContext
```

Comment:
I like how the SuppressWarnings annotation is 4 times longer than the explicit type signature :)